examples/js/e-commerce-umd/index.html

```diff
@@ -17,13 +17,10 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Hind:400,600|Open+Sans:300,400,700&display=swap"
     />
-    <!--
-      Do not use @8 in production, use a complete version like x.x.x, see website for latest version:
-      https://www.algolia.com/doc/guides/building-search-ui/installation/js/#load-the-styles
-    -->
+
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/instantsearch.css@8/themes/reset-min.css"
+      href="/packages/instantsearch.css/themes/reset-min.css"
```

Comment:
this example now won't work on codesandbox anymore, but it already didn't so that's ok